students/hellosputnik/main.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+)
+
+type Problem struct {
+	question string
+	answer   string
+}
+
+type Quiz struct {
+	problems []Problem
+	score    int
+}
+
+type Settings struct {
+	filename  *string
+	timeLimit *int
+}
+
+func main() {
+	quiz := Quiz{}
+	settings := Settings{}
```

Comment:
Might be a personal preference, but when creating the zero value of a type I prefer to do:

```go
var settings Settings
```

Reply:
bonus points for

```
var (
	quiz Quiz
	settings Settings
)
```